explorer/app/hooks/useFetchEntities.ts

```diff
@@ -65,10 +63,11 @@ export const useFetchEntities = (value?: ListModel): UseEntityListResponse => {
     data: apiData,
     isLoading: apiIsLoading,
     run,
+    isIdle,
   } = useAsync<ListResponseType>();
 
   useEffect(() => {
-    if ((!staticLoad || isDevelopment()) && !isSSR()) {
```

Comment:
A lot has changed since this check was created, and I noticed that it doesn't seem to make sense anymore. So I tried removing it and tested every config (I'm having some issues with anvil and hca's API, so could do the complete test for these configs, but it's working for all the others.)

Reply:
Thank you!